api/service/legacysync/syncing.go

```diff
@@ -844,7 +873,7 @@ func (ss *StateSync) generateNewState(bc *core.BlockChain, worker *worker.Worker
 		if block == nil {
 			break
 		}
-		err = ss.UpdateBlockAndStatus(block, bc, worker, false)
+		err = ss.UpdateBlockAndStatus(block, bc, worker, true)
```

Comment:
why change to true for verifyAllSig?

Reply:
These blocks were designed to be obtained from DNS node's register/push (DownloaderRequest_REGISTER). And these blocks can possibly be the last block of one sync loop after dealing with "commonBlocks". So we need to verify the signature and flush commitSigAndBitmap to db. The amount of this signature verification is not large at all.

Reply:
This reminds to to also update NEWBLOCK request. Thanks.

Reply:
NEWBLOCK push request also updated with Signature. So this value can be safely set to true. This is to verify the new block signature pushed from DNS nodes. The amount of this blocks is not much compared with `commonBlocks`.

Reply:
Updated logic to fix errors. Need two bools to check the signature values. Please check the latest code again. Thanks!